numba_dpex/tests/kernel_tests/test_kernel_launch_params.py

```diff
@@ -22,9 +22,10 @@ def vecadd(a, b, c):
 
 
 def test_1D_global_range_as_int():
-    k = vecadd[10]
-    assert k._global_range == [10]
-    assert k._local_range is None
+    with pytest.deprecated_call():
```

Comment:
Why do we need `pytest.deprecated_call()`? I think we should remove these tests or rewrite with the new calling syntax.

Reply:
Yes, I agree. But that should be a separate PR. I just wanted to silence the deprecation warning as it was flooding the logs.